sale_purchase_requisition/models/sale_order.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 Moduon Team S.L. <info@moduon.team>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    purchase_agent = fields.Many2one(
+        comodel_name="res.users", string="Comercial compra"
+    )
```

Comment:
It's already an user_id on the sale, why to add a new one?

If it's necessary, change the name to be in English

Reply:
This field is needed because the purchase agent might be different from the salesman.

However indeed it must change the name. Also, the field must follow the convention on naming:

```suggestion
    purchase_agent_id = fields.Many2one(
        comodel_name="res.users", string="Purchase agent"
    )
```